src/test/java/no/unit/nva/cristin/projects/FetchCristinProjectsTest.java

```diff
@@ -84,47 +84,47 @@ void handlerReturnsExpectedBodyWhenRequestInputIsValid(String expected) throws I
         cristinApiClientStub = spy(cristinApiClientStub);
-        doThrow(new IOException()).when(cristinApiClientStub).getProject(any(), any());
+        doThrow(BadGatewayException.class).when(cristinApiClientStub).getProject(any(), any());
         handler = new FetchCristinProjects(cristinApiClientStub, environment);
+
         GatewayResponse<ProjectsWrapper> response = sendDefaultQuery();
-        var expected = IoUtils.stringFromResources(Path.of(API_RESPONSE_NON_ENRICHED_PROJECTS_JSON));
-        assertEquals(OBJECT_MAPPER.readTree(expected), OBJECT_MAPPER.readTree(response.getBody()));
+
+        assertEquals(HttpURLConnection.HTTP_OK, response.getStatusCode());
+        assertEquals(MediaType.APPLICATION_JSON, response.getHeaders().get(HttpHeaders.CONTENT_TYPE));
     }
 
     @Test
-    public void handlerReturnsOkWhenInputContainsTitleAndLanguage() throws Exception {
+    void handlerReturnsOkWhenInputContainsTitleAndLanguage() throws Exception {
         GatewayResponse<ProjectsWrapper> response = sendDefaultQuery();
         assertEquals(HttpURLConnection.HTTP_OK, response.getStatusCode());
     }
 
     @Test
-    public void handlerIgnoresErrorsWhenTryingToEnrichProjectInformation() throws Exception {
+    void handlerThrowsInternalErrorWhenQueryingProjectsFails() throws Exception {
         cristinApiClientStub = spy(cristinApiClientStub);
-        doThrow(new IOException()).when(cristinApiClientStub).getProject(any(), any());
+        doThrow(new RuntimeException()).when(cristinApiClientStub).queryAndEnrichProjects(any());
         handler = new FetchCristinProjects(cristinApiClientStub, environment);
 
         GatewayResponse<ProjectsWrapper> response = sendDefaultQuery();
 
-        assertEquals(HttpURLConnection.HTTP_OK, response.getStatusCode());
-        assertEquals(MediaType.APPLICATION_JSON, response.getHeaders().get(HttpHeaders.CONTENT_TYPE));
+        assertEquals(HttpURLConnection.HTTP_INTERNAL_ERROR, response.getStatusCode());
+        assertEquals(APPLICATION_PROBLEM_JSON, response.getHeaders().get(HttpHeaders.CONTENT_TYPE));
     }
 
     @Test
-    public void handlerThrowsInternalErrorWhenQueryingProjectsFails() throws Exception {
+    void handlerReturnsNonEnrichedBodyWhenEnrichingFails() throws Exception {
         cristinApiClientStub = spy(cristinApiClientStub);
-        doThrow(new IOException()).when(cristinApiClientStub).queryAndEnrichProjects(any());
+        doThrow(BadGatewayException.class).when(cristinApiClientStub).getProject(any(), any());
         handler = new FetchCristinProjects(cristinApiClientStub, environment);
-
         GatewayResponse<ProjectsWrapper> response = sendDefaultQuery();
-
-        assertEquals(HttpURLConnection.HTTP_INTERNAL_ERROR, response.getStatusCode());
-        assertEquals(APPLICATION_PROBLEM_JSON, response.getHeaders().get(HttpHeaders.CONTENT_TYPE));
+        var expected = IoUtils.stringFromResources(Path.of(API_RESPONSE_NON_ENRICHED_PROJECTS_JSON));
```

Comment:
I don't disagree with the use of var, but I guess we should be consistent across the project.